docs/grammar/BallerinaParser.g4

```diff
@@ -333,7 +341,7 @@ breakStatement
 
 // typeName is only message
 forkJoinStatement
-    : FORK LEFT_BRACE workerDeclaration* RIGHT_BRACE joinClause? timeoutClause?
+    : FORK LEFT_BRACE connectorVarDefStatement* workerDeclaration* RIGHT_BRACE joinClause? timeoutClause?
```

Comment:
You don't need put  'connectorVarDefStatement' inside fork/join statements. I don't think this is correct.